src/routes/townhalls/townhalls.test.ts

```diff
@@ -25,6 +25,15 @@ describe('townhall routes', () => {
             expect(status).toStrictEqual(200);
         });
     });
+    /// test returns 500 because townhall id is technically null until a townhall create post request is made
+    describe('/townhalls/:townhallId/bills', () => {
+        it('should be status 500', async () => {
+            const { status } = await request(app).get(
+                `/townhalls/${new ObjectID().toString()}/bills`
```

Comment:
Refer to this https://github.com/c2c-project/prytaneum-notifications/blob/master/src/routes/index.test.ts#L66-L83 for an example of how to setup mock data.